controllers/chat/group/leaveGroupMember.js

```diff
@@ -16,6 +16,13 @@ const leaveGroupMember = async (req, res) => {
       const {newChannelName, betterChannelMember, betterChannelMemberObject, updatedChannel} =
         await BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse, true);
 
+      await channel.sendMessage({
+        text: 'Leave group',
+        type: 'system',
+        user_id: userId,
+        userAffectedId: userId
+      });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code sends a system message when a user leaves the group. However, it's not clear what the 'Leave group' message means. It would be more informative if the message included the username of the person who left the group. This way, other members can easily understand who left the group.

```diff
-      await channel.sendMessage({
-        text: 'Leave group',
-        type: 'system',
-        user_id: userId,
-        userAffectedId: userId
-      });
+      const userName = /* fetch the username using userId */;
+      await channel.sendMessage({
+        text: `${userName} has left the group`,
+        type: 'system',
+        user_id: userId,
+        userAffectedId: userId
+      });
```
Please replace `/* fetch the username using userId */` with the actual code to get the username from the userId.


<!-- This is an auto-generated comment by OSS CodeRabbit -->